packages/mask/src/plugins/WalletService/services/wallet/password.ts

```diff
@@ -61,7 +75,8 @@ export async function verifyPasswordRequired(unverifiedPassword: string) {
 
 export async function changePassword(oldPassword: string, newPassword: string) {
     validatePasswordRequired(newPassword)
-    validatePasswordRequired(oldPassword)
+    const isVerify = await verifyPassword(oldPassword)
+    if (!isVerify) throw new Error('Incorrect payment password.')
```

Comment:
```suggestion
    const verified = await verifyPassword(oldPassword)
    if (!verified) throw new Error('Incorrect payment password.')
```

Reply:
Did you mean `verifyPassword(newPassword)`.

Reply:
No